src/providers/data_providers/googlePlacesBusinessImageProvider.ts

```diff
@@ -0,0 +1,25 @@
+import { BusinessInfoInput, Geocode } from '../../models/data_models/types';
+import { BaseBusinessImageProvider } from './baseBusinessImageProvider';
+import { GooglePlacesApiUtils } from '../../utils/googlePlacesApiUtils';
+import { Image, ImageProviderName } from '../../models/data_models/image';
+
+export class GooglePlacesBusinessImageProvider extends BaseBusinessImageProvider {
+	public async getBusinessImages(businessInfo: BusinessInfoInput): Promise<Array<Image>> {
+		const googlePlacesApi = await new GooglePlacesApiUtils(this.config).init();
+		return googlePlacesApi
+			.getPlaceID(businessInfo.address, businessInfo.name)
+			.then((id) => googlePlacesApi.getPlaceDetails(id))
+			.then((photoRefs) =>
+				Promise.all(photoRefs.map((photoRef) => googlePlacesApi.getPhoto(photoRef)).filter((x) => x))
+			)
+			.then((photoUrls) =>
+				photoUrls.map(
+					(url: any) =>
```

Comment:
This `: any` type annotation shouldn't be necessary. If `url` isn't explicitly a string, there is probably another underlying issue (possible the `.filter` from the previous `.then` not working as expected, and not removing the `undefined` case from the results of `googlePlacesApi.getPhoto()`).